crypto-keys/src/main/java/com/palantir/crypto2/keys/serialization/KeyMaterials.java

```diff
@@ -43,6 +45,15 @@
 
     private static final Logger log = LoggerFactory.getLogger(KeyMaterials.class);
     private static final Map<Integer, ? extends KeySerializer> SERIALIZERS = KeySerializers.getSerializers();
+    private static final CryptoRandom random = getRandom();
+
+    private static CryptoRandom getRandom() {
+        try {
+            return CryptoRandomFactory.getCryptoRandom();
```

Comment:
It'll default to using an `OpenSslCryptoRandom`, which'll use hardware instructions if possible, and if not, then it looks like it'll use `/dev/random` based on my reading of https://wiki.openssl.org/index.php/Random_Numbers. If native libs aren't loaded, it'll fall back to using a `JavaCryptoRandom`, which uses a `java.security.SecureRandom` under the hood.